docs/examples/notebooks/06-chaotic_attractor.ipynb

```diff
@@ -0,0 +1,265 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Lorenz Chaotic Attractor\n",
+    "\n",
+    "This example show how a classical chaotic dynamical system (the Lorenz \"butterfly\" attractor) can be implemented in a neural population. The dynamical equations for this attractor are:\n",
+    "$$\n",
+    "\\begin{align}\n",
+    "    \\dot{x}_0 &= \\sigma(x_1 - x_0) \\\\\n",
+    "    \\dot{x}_1 &= x_0(\\rho - x_2) - x_1 \\\\\n",
+    "    \\dot{x}_2 &= x_0 x_1 - \\beta x_2\n",
+    "\\end{align}\n",
+    "$$\n",
+    "\n",
+    "Since $x_2$ is approximately centered around $\\rho$, and because NEF ensembles are typically optimized to represent values within a pre-defined radius of the origin, we substitute $x'_2 = x_2 - \\rho$, yielding these equations:\n",
+    "$$\n",
+    "\\begin{align}\n",
+    "    \\dot{x}_0 &= \\sigma(x_1 - x_0) \\\\\n",
+    "    \\dot{x}_1 &= -x_0 x'_2 - x_1 \\\\\n",
+    "    \\dot{x}'_2 &= x_0 x_1 - \\beta(x'_2 + \\rho) - \\rho\n",
+    "\\end{align}\n",
+    "$$\n",
+    "\n",
+    "As with the previous oscillator examples, an exponential synapse is being used to implement these dynamics, meaning that the neural feedback matrix is:\n",
+    "$$\n",
+    "\\mathbf{A}' = \\tau\\mathbf{A} + \\mathbf{I}\n",
+    "$$\n",
+    "\n",
+    "Because the Lorenz attractor has no input signal $u$, this in turn means that the neural feedback function can be computed as:\n",
+    "$$\n",
+    "\\begin{align}\n",
+    "    f_{fdbk}(x) &= \\mathbf{A}' x \\\\\n",
+    "    &= \\tau\\mathbf{A}x + x \\\\\n",
+    "    &= \\tau\\dot{x} + x \\\\\n",
+    "    &= \\begin{bmatrix}\n",
+    "           \\tau\\dot{x}_0 + x_0 \\\\\n",
+    "           \\tau\\dot{x}_1 + x_1 \\\\\n",
+    "           \\tau\\dot{x}'_2 + x_2\n",
+    "       \\end{bmatrix}\n",
+    "\\end{align}\n",
+    "$$\n",
+    "\n",
+    "For more information, see http://compneuro.uwaterloo.ca/publications/eliasmith2005b.html “Chris Eliasmith. A unified approach to building and controlling spiking attractor networks. Neural computation, 7(6):1276-1314, 2005.”"
```

Comment:
Do we want to embed the link into the title, or another keyword (e.g. 'this publication') instead of rendering the raw URL?